doc/hardware/porting/board_porting.rst

```diff
@@ -242,6 +242,36 @@ followed by trial and error. If you want to understand details, you will need
 to read the rest of the devicetree documentation and the devicetree
 specification.
 
+Board extensions
+****************
+
+Board extensions are board fragments that may be present in any board root
+folder, under ``${BOARD_ROOT}/extensions/${BOARD}``. The board extension
```

Comment:
I think this line must be updated, cause it is not `BOARD` that must be used here.

For example, to extend a board like `nrf9160dk_nrf9160_ns` the extension must be placed in a folder name `nrf9160dk_nrf9160` (notice the missing `_ns`), but `BOARD` variable contains `nrf9160dk_nrf9160_ns`.

I'm not having a text proposal on how to make this description clearer.